influxql/functions.go

```diff
@@ -395,12 +397,14 @@ func NewIntegerCumulativeSumReducer() *IntegerCumulativeSumReducer {
 	}
 }
 
+// AggregateInteger aggregates a point into the reducer
 func (r *IntegerCumulativeSumReducer) AggregateInteger(p *IntegerPoint) {
 	r.curr.Value += p.Value
 	r.curr.Time = p.Time
 	r.curr.Nil = false
 }
 
+// Emit protects of null and emits a point
```

Comment:
Fullstop needed. `Emit protects of null and emits a point` doesn't make sense. How about `Emit emits the next point from the reducer.`?